lib/CMakeLists.txt

```diff
@@ -1,6 +1,3 @@
-# Likewise, absolutely no idea what I'm doing.
-# YOLO, I guess.
+add_library(disccord SHARED client.cpp)
```

Comment:
Assuming disccord is a shared lib. Change "SHARED" to "STATIC" if that ain't the case. You can also add `add_library(disccord_s STATIC client.cpp)` if you want both shared and static libs